tests/Exercise/BettysBikeShopTest.elm

```diff
@@ -0,0 +1,185 @@
+module Exercise.BettysBikeShopTest exposing (..)
+
+import Comment exposing (Comment, CommentType(..))
+import Dict
+import Exercise.BettysBikeShop as BettysBikeShop
+import Review.Rule exposing (Rule)
+import Review.Test
+import Test exposing (Test, describe, test)
+import TestHelper
+
+
+rules : List Rule
+rules =
+    [ BettysBikeShop.hasFunctionSignatures ]
+
+
+exemplar : Test
```

Comment:
I think we can remove this test once the import statement is removed

Reply:
> > But there is no need to import String since it's already imported by default. I was actually thinking of implementing an elm-review rule about that. Is this worth a PR for removing it, or ignoring is fine?
> 
> I think it is probably best to remove this. We should just check that the exercise still achieves its goals though.

The stated goals in `design.md` are
- Know how to import functions from modules. 
- Know how to export functions for use by other modules.
- Know how to define type annotations for functions.
- Know how to work with integers and floats.
- Know how to use basic string operations.

I'm not sure if the first 2 are achieved...

## Know how to import functions from modules

The introduction mentions that qualified imports are preferred 
```elm
import List                            -- List.map, List.foldl
import List as L                       -- L.map, L.foldl
```
So `import String` would be fine, except for the fact that that particular one is not necessary. Now that I think about it more, maybe it's fine to leave it for the sake of learning. 
But in this case, should I add a check that `String` was actually imported? 
We could also ask the students to give is an alias `S` or something and add a check on that.

## Know how to export functions for use by other modules

The concept introduction mentions two types of importing
```elm
module Calculator exposing (add)
module Calculator exposing (..)
```
But the stub files already does it, the students don't need to do anything.
Maybe we could change the stub to only have one function exported? 
Or  nothing at all, even if it doesn't compile?

Reply:
It's a bit difficult to force students to import a module that isn't already in the prelude when we are only in first or second concept exercise. Yet it's important to talk about how imports work, for student to understand why we call `String.fromFloat` in that way.

For exports, I suppose we could start the template file with only one of the two imports. This way, they still have something to do, but it's super easy.
```elm
module BettysBikeShop exposing (penceToPounds)
```

Reply:
I think there is maybe a longer chat to have about other parts of the Bettys Bike Shop exercise. Shall we park that conversation for now and create an issue to track it? And then we can move on with the PR?
I'm on holiday for a couple of weeks by the way (back on May 1st) so please feel free to make decisions without me.

Reply:
I think we have a handle on things now. Here is my plan:

For imports, we don't change anything in the exercise, but I add a rule in the analyzer that says something like "Please use `import String`. The `String` module doesn't actually need to be called because it's there by default, but importing module is important and this is the place to practice it".

For exports, we change the stub have one function exported so it will not compile. We add one task (in first position) with only one test `test "both functions are exported" <| \() -> Expect.pass`. Basically, if it compiles it means they exported the function therefore it passes. For this there is no need for the analyzer.

That means both things can be done independently! 
How does that sound?

@ceddlyburge enjoy your vacation! We'll try not to burn the track to the ground :)

Reply:
sounds good! enjoy @ceddlyburge !